src/OrchardCore.Modules/OrchardCore.Lucene/Shapes/LuceneContentPickerShapeProvider.cs

```diff
@@ -3,9 +3,11 @@
 using OrchardCore.DisplayManagement;
 using OrchardCore.DisplayManagement.Descriptors;
 using OrchardCore.DisplayManagement.Shapes;
+using OrchardCore.Modules;
 
 namespace OrchardCore.Lucene
 {
+    [Feature("OrchardCore.Lucene.ContentPicker")]
     public class LuceneContentPickerShapeProvider : IShapeAttributeProvider
```

Comment:
Registered when the feature is enabled but as a feature type needs to be tied to the related feature

Reply:
Rename this as LuceneContentPickerShape

Reply:
Or merge your PR and I'll merge on top of mine

Reply:
Wait I'll merge your branch on top of my PR instead

Reply:
No, good to keep it `IShapeAttributeProvider` because it's really a provider because at it registered as is through `AddShapeAttributes` that does

        public static IServiceCollection AddShapeAttributes<T>(this IServiceCollection services) where T : class, IShapeAttributeProvider
        {
            services.AddScoped<T>();
            services.AddScoped<IShapeAttributeProvider, T>();

            return services;
        }

Reply:
I know but we always call the classes without the Provider part. SearchShapes, PagerShapes ...